color_refinement.py

```diff
@@ -78,6 +79,60 @@ def color_refine(coloring: "Coloring") -> "Coloring":
     return coloring
 
 
+def fast_color_refine(graph: "Graph"):
+    # takes initial coloring
+    qlist = DoubleLinkedList()
+    color_map = {}
+    neigh_map = {}
+    #TODO: Replace this with better things, because it's now just used to get the amount of colors
+    # because I did not figure out how to do this with the Coloring class yet
+    # Neigh_map might not be neccessary
+    for v in graph.vertices:
+        if v.colornum not in color_map.keys():
+            color_map[v.colornum] = []
+        color_map[v.colornum].append(v)
+        if v not in neigh_map.keys():
```

Comment:
Wat doet dit?

Reply:
Want momenteel is `neigh_map` leeg

Reply:
Dit deed eigenlijk een map maken van alle vertices naar zn neighbours. De `if .. not in ` was niet nodig.